rclpy/rclpy/__init__.py

```diff
@@ -12,12 +12,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import rclpy._rclpy__rmw_opensplice_cpp as _rclpy
+import importlib
 
 from rclpy.node import Node
+from rclpy.vendors import rmw_implementations
+
+global _rclpy
+try:
+    _rclpy
+except NameError:
+    _rclpy = None
+if _rclpy is None:
+    for rmw_implementation in rmw_implementations:
+        module_name = '._rclpy__{rmw_implementation}'.format(
+            rmw_implementation=rmw_implementation,
+        )
+        rmw_module = importlib.import_module(module_name, package='rclpy')
+        if rmw_module is not None:
+            _rclpy = rmw_module
+            break
```

Comment:
How about an environment variable, but still be able to switch the RMW implementation programmatically? That way it could be changed outside Python, but also from inside Python.